ITEM.tsv

```diff
@@ -492,7 +492,7 @@ ITEM_20150317_000491	A very simple weapon for anyone to use without difficulty.
 ITEM_20150317_000492	Wooden Club
 ITEM_20150317_000493	Even an ordinary wooden stick can become a holy item if wielded by a Priest.
 ITEM_20150317_000494	Dunkel Iron Club
-ITEM_20150317_000495	A weapon of at least of this caliber is necessary to carry out the Goddess' punishment.
+ITEM_20150317_000495	A weapon, at least of this caliber, is necessary to carry out the Goddess's punishment.
```

Comment:
When a noun ends in an S and has possession of a noun like a trait or object , add only one apostrophe. So it should be Goddess' not Goddess's.